docs/TESTING.md

```diff
@@ -0,0 +1,55 @@
+# Testing
+
+Testing is an integral part of creating new features and maintaining the software.
+
+## Automated testing
+
+A [few actions](https://github.com/Automattic/vip-design-system/tree/trunk/.github/workflows) are automatically run via Github Actions when a pull request is created or updated.
+
+### Linting
+
+We run the following checks:
+
+- linting (`npm run lint`)
+- format checking (`npm run format:check`)
+- type checks (`npm run check-types`)
+
+### Unit tests
+
+Located in [test/](https://github.com/Automattic/vip-design-system/tree/trunk/test).
```

Comment:
Only two files are present there and they do not seem to be tests. Feels like this text needs to explain what these files are and where the actual unit tests are located.